app/helpers/hyku_addons/notes_tab_form_helper.rb

```diff
@@ -3,7 +3,7 @@ module HykuAddons
   module NotesTabFormHelper
     def form_tabs_for(form:)
       if Flipflop.enabled?(:notes_tab_form)
-        super + ['notes']
+        super + ["notes"]
```

Comment:
I thought your suggestion was to use double quotes only in contatenation

Reply:
The other way around, only use single quotes inside of a double quotes string where you are performing a conditional. 

There is no speed different between the two and double quotes allow variables to be parsed, so it means we would only use one two of quote, rather than two types if we used single quotes.